pkgs/by-name/ki/kitget/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kiget";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "adamperkowski";
+    repo = "kitget";
+    tag = "v${version}";
```

finalAttrs is generally preferred to rec
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
  pname = "kiget";
  version = "0.0.1";

  src = fetchFromGitHub {
    owner = "adamperkowski";
    repo = "kitget";
    tag = "v${finalAttrs.version}";
```